pkg/status/dist/templates/logsagent.tmpl

```diff
@@ -10,6 +10,13 @@ Logs Agent
   Logs Agent is not running
 {{- end }}
 
+{{- if .endpoints }}
+
+  {{- range $endpoint := .endpoints }}
```

Comment:
Would probably nice to add a header for this section

Reply:
WDYT @NBParis ?

Reply:
Not sure about this, as this should be the first thing we show in the status, having the Log section starting with a descriptive sentence should be enough without requiring another header.

Reply:
sounds fair